### PR TITLE
Home work 20

### PR DESCRIPTION
**Pull request by chernyakma**

### CODE REVIEW

File: src/test/java/PlayListTests.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaylistTests extends BaseTest{
```
Comment (Berichicko) [inline src/test/java/PlayListTests.java:6]: The code is not compiled! Please finish the homework!You got this problem, please correct it. 

image

Comment (chernyakma) [reply src/test/java/PlayListTests.java:6]: Screenshot (1)
Hi Alex! Can you check me again, please. It looks like everything works for me))

Comment (Berichicko) [reply src/test/java/PlayListTests.java:6]: Hello

Please, re-submit homework 20 for review

Comment (Berichicko) [reply src/test/java/PlayListTests.java:6]: The code is not compiled! Please finish the homework!You got this problem, please correct it. 

image

HINT

You forgot to make a commit and push the fixes to GitHub

image

Comment (chernyakma) [reply src/test/java/PlayListTests.java:6]: Sorry. There were stashed. Now I committed all changes.

**Pull request closed without merging**